circleguard/frametime_graph.py

```diff
@@ -94,6 +94,8 @@ def plot_with_break(self, frametimes):
     def _conversion_factor(self, replay):
         if Mod.DT in replay.mods:
```

Comment:
I guess it would be fine (and a little more performant) if you just add it to the first conditional.
```suggestion
        if Mod.DT or Mod.NC in replay.mods:
```